lsmb-request.pl

```diff
@@ -19,7 +19,7 @@ =head1 COPYRIGHT
 =cut
 
 package LedgerSMB::Handler;
-
+use lib '.';
```

Comment:
rather than adding '.' as a relative path, can we explicitly add the Current Working Directory?